### ISSUE TITLE
DB error with 1.7 DR3

### ISSUE THREAD

**Issue by leovipin1** [osTicket/osTicket-1.7#97]
Hi,

I am getting the below mentioned DB Error when I create a ticket.

DB Error #1054

[SELECT m.staff_id FROM ost7_staff m WHERE m.dept_id=1 AND s.staff_id IS NOT NULL ORDER BY s.lastname, s.firstname]

Unknown column 's.staff_id' in 'where clause'

Kindly look into this and also let us know the date when the stable version of 1.7 will be available.

**Comment by bgarlock** [comment]
I am also seeing this error - is there a fix for it yet?

**Comment by bgarlock** [comment]
FYI:  To get rid of this error, I edited 'class.dept.php' and changed the following function:

```
    if(!$this->members && $this->getNumStaff()){
        $sql='SELECT staff_id FROM '.STAFF_TABLE.' '
            .'WHERE dept_id='.db_input($this->getId())
            .' AND staff_id IS NOT NULL '
            .'ORDER BY lastname, firstname';
        if(($res=db_query($sql)) && db_num_rows($res)){
            while(list($id)=db_fetch_row($res))
                if($staff= Staff::lookup($id) && $staff->isAvailable())
                    $this->members[]= $staff;
        }
```

**Comment by bgarlock** [comment]
Actually, this fixes it :-)

https://github.com/putneyj/osTicket-1.7/commit/169700f9cbdf596d3e8196ca718eb3eb64aed8dd

**Comment by greezybacon** [comment]
This was fixed thanks to putneyj in issue #115. Thanks for reporting
